### PR TITLE
Update recommendations.md

### PR DESCRIPTION
**Pull request by julian-cable**
@celestehorgan Here is my first draft proposal for a term template.

### CODE REVIEW

**Comment by carlaquinn** [comment]
@julian-cable this looks very thorough and usable. I really like it.

**Review by celestehorgan — APPROVED** [review]
Awesome! Thanks @julian-cable

File: ws-language/templates/recommendations.md [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
+| Term H | Use with caution | adjective, noun | Do not use in a context where ... [Justification] |
+| Term I | Do not use | noun | Do not use. Use "[preferred-term]". [Justification] |
+| Term J | Do not use | noun | Do not use. Use another term, such as "[option-1]" or "[option-2]". [Justification] |
+
+Example entries:
+| Term | Status | Part of speech | Usage note | 
+| --- | --- | --- | --- |
+| Indigenous | Preferred | adjective | Write as shown, with initial uppercase I, to refer to people who are native to a region or country. |
+| master | Use with caution | adjective, noun | Do not use when it is paired with "slave", explicitly or implicitly; such use evokes the dehumanizing practice of slavery. Use alternative terms such as "primary" and "secondary", or "[alternative-term1]" and "[alternative-term2]" , or "[alternative-term3]" and "[alternative-term4]". However, use of "master" is acceptable in other contexts, such as to refer to mastery of a skill. |
+| segregation | Do not use | noun | Do not use. Use "separation" or "segmentation". This term refers to a process that violates human rights by limiting access to privileges and rights such as education and voting. |
```
Comment (markcmiller86) [inline ws-language/templates/recommendations.md:30]: In this example verbiage, which is good to have BTW, I would opt not to include some of the candidate replacement terms and instead use the proxy text `alternative-term-1` and `alternative-term-2`. I think its far too likely and easy that others stumbling here will read this less as an example of how to assert guidance and more as actual guidance.

**Comment by julian-cable** [comment]
@markcmiller86 To your previous comments (just my opinion rather than that of INI):

1. Although the primary need is to provide guidance for terms that are not to be used, in some cases INI might have a need to indicate what is the preferred term to use or a preferred way of writing a term. For example, a suggested replacement for "white hat hacker" might be "offensive security researcher". The latter term might not be widely known, so if it is adopted, INI might usefully add a _Preferred_ entry for it, to define the term and provide any justification about why it was chosen.
2. There might not be many such cases, but guidance might differ depending on part of speech. For example, to _master_ a skill (verb) might be allowed, whereas master (noun, adjective) might not be allowed in other cases. 
3. _Use with caution_ has a different meaning from _avoid where possible_. The former means not to use the term except for in a specific allowed case, or conversely that it's OK to use the term except in a specific disallowed case. A term that is never allowed would be _Do not use_.
4. Yes, _must_ is a requirement and _should_ is a recommendation. I would avoid _may_, which indicates permission, and instead use _might_ to indicate possibility.

**Pull request merged**